LoRaEngine/LoraKeysManagerFacade/LoRaDeviceCacheRedisStore.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoraKeysManagerFacade
+{
+    using System;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Extensions.Configuration;
+    using StackExchange.Redis;
+
+    public class LoRaDeviceCacheRedisStore : ILoRaDeviceCacheStore
+    {
+        private IDatabase redisCache;
+
+        public LoRaDeviceCacheRedisStore(ExecutionContext context)
+        {
+            var config = new ConfigurationBuilder()
```

Comment:
Suggestion: this configuration could be built once, for RegistryManager and RedisStore

Reply:
True. Will change that. There is another thing I noticed: we add the option to reload on all of these, if the file changes, but we do not really do anything if we reload: 
1. are we ever going to need to reload (do we support updating the config file)
2. if 1 is true, we should add code to actually refresh the various objects we build from it
3. if 1 is false, we should remove the reload option.

Reply:
AFAIK if the configuration file changes the Azure Function will be restarted. It might be useful for local development, but I wouldn't invest any time on it.

Reply:
Ok. Then I'll disable the re-load flag.